flavor.go

```diff
@@ -146,6 +151,9 @@ func (f Flavor) PrepareInsertIgnore(table string, ib *InsertBuilder) {
 	case SQLite:
 		// see https://www.sqlite.org/lang_insert.html
 		ib.verb = "INSERT OR IGNORE"
+	case ClickHouse:
+		// see https://clickhouse.tech/docs/en/sql-reference/statements/insert-into/
+		ib.verb = "INSERT"
```

Comment:
The verb should be `INSERT INTO`. And it also should be `INSERT INTO` in PostgreSQL flavor -- it should be an aged bug.